metecho/api/jobs.py

```diff
@@ -515,14 +515,19 @@ def populate_github_users(repository, *, originating_user_id):
         repo_id = repository.get_repo_id(user)
         repo = get_repo_info(user, repo_id=repo_id)
         repository.refresh_from_db()
-        repository.github_users = [
-            {
-                "id": str(collaborator.id),
-                "login": collaborator.login,
-                "avatar_url": collaborator.avatar_url,
-            }
-            for collaborator in repo.collaborators()
-        ]
+        repository.github_users = list(
+            sorted(
+                [
+                    {
+                        "id": str(collaborator.id),
+                        "login": collaborator.login,
+                        "avatar_url": collaborator.avatar_url,
+                    }
+                    for collaborator in repo.collaborators()
+                ],
+                key=lambda x: x["login"],
```

Comment:
@wlonk This appears to be sorting case-sensitive, where I think we want case-insensitive.

<img width="639" alt="Screenshot 2020-03-28 13 59 38" src="https://user-images.githubusercontent.com/552316/77830083-62cea200-70fc-11ea-8a82-c685ced3b88e.png">

Reply:
@wlonk Also, could we sort the list of `project.github_users`?

Reply:
The backend never sets `project.github_users`, so the preferred way would be for the frontend to sort it when it sets it.

Reply:
Ok, I can do that. I thought maybe sorting in the serializer would be simplest.